### PR TITLE
improved outro notebook

### PR DESCRIPTION
**Pull request by fkiraly**
Made sume suggested changes, @benHeid 

* streamlining bullet point presentation
* shortening the double pipeline
* sharpening the points of what the graph pipeline adds - not e2e but simplicity (less nested/convoluted)
* roadmap sharpened
* summary section
* up-to-date "join us"

### CODE REVIEW

**Comment by fkiraly** [comment]
*important*, @benHeid - the data is not in the repo. The notebook does not run in binder. Please fix that asap!

**Comment by benHeid** [comment]
Thank you for the changes, I am fine with these and sorry for not pinging on the data PR...

**Review by benHeid — APPROVED** [review]

File: notebooks/04-non-sequential.ipynb [hunk 1]
```diff
@@ -2,73 +2,63 @@
  "cells": [
```
Comment (benHeid) [inline notebooks/04-non-sequential.ipynb:2]: <p>Can you make the next bullet point smaller?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/sktime/sktime-tutorial-pydata-global-2022/pull/10/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Comment (benHeid) [reply notebooks/04-non-sequential.ipynb:2]: It is only big in reviewnb..

Comment (fkiraly) [reply notebooks/04-non-sequential.ipynb:2]: does that mean, I should not make it smaller?

You have write access, feel free to change it.

Comment (benHeid) [reply notebooks/04-non-sequential.ipynb:2]: Leave it as it is. It loooks good :)

**Pull request merged**